docs/operating-scylla/procedures/cluster-management/arbiter-dc.rst

```diff
@@ -0,0 +1,50 @@
+=========================================================
+Preventing Quorum Loss in Symmetrical Multi-DC Clusters
+=========================================================
+
+ScyllaDB requires at least a quorum (majority) of nodes in a cluster to be up
+and communicate with each other. A cluster that loses a quorum can handle reads
+and writes of user data, but cluster management operations, such as schema and
+topology updates, are impossible.
+
+In clusters that are symmetrical, i.e., have two (DCs) with the same number of
+nodes, losing a quorum may occur if one of the DCs becomes unavailable.
+For example, if one DC fails in a 2-DC cluster where each DC has three nodes,
+only three out of six nodes are available, and the quorum is lost.
+
+Adding another DC would mitigate the risk of losing a quorum, but it comes
+with network and storage costs. To prevent the quorum loss with minimum costs,
+you can configure an arbiter (tie-breaker) DC.
+
+An arbiter DC is a datacenter with a :doc:`zero-token node </architecture/zero-token-nodes>`
+-- a node that doesn't replicate any data but is only used for Raft quorum
+voting. An arbiter DC maintains the cluster quorum if one of the other DCs
+fails, while it doesn't incur extra network and storage costs as it has no
+user data.
+
+Adding an Arbiter DC
+-----------------------
+
```

Comment:
I think we should mention somewhere that zero-token nodes are ignored by drivers, so there is no need to change the load balancing policy on the clients. Basically, [Configure the Client not to Connect to the New DC](https://opensource.docs.scylladb.com/stable/operating-scylla/procedures/cluster-management/add-dc-to-existing-dc.html#configure-the-client-not-to-connect-to-the-new-dc) can be skipped.

Reply:
Maybe we can also note that the Arbiter DC has RF=0 for all user keyspaces and all these keyspaces must be updated just like in case of normal DC. I'm not sure how obvious it is.

Another difference is that `nodetool rebuild` and `nodetool repair` can be skipped. Zero-token nodes never store replicated data, so these operations do nothing. Again, I'm not sure if it's worth documenting.

Reply:
> Another difference compared to adding normal DC is that nodetool rebuild and nodetool repair can be skipped. Zero-token nodes never store replicated data, so these operations do nothing. Again, I'm not sure if it's worth documenting.

I think it's worth mentioning, but not on arbiter DC page, but on zero-token nodes page.

Reply:
If we do it like this, then we can additionally mention `nodetool cleanup`. It also does nothing for zero-token nodes.

Reply:
> If we do it like this, then we can additionally mention nodetool cleanup. It also does nothing for zero-token nodes.

IMHO, Too much info in this case.